mysql-test/lsan.supp

```diff
@@ -24,3 +24,7 @@
 leak:Perl_safesyscalloc
 leak:Perl_safesysmalloc
 leak:Perl_safesysrealloc
+leak:Perl_savesharedpv
+leak:Perl_Slab_Alloc
+leak:Perl_newUNOP_AUX
+leak:/usr/bin/perl
```

Comment:
Please also add
```
diff --git a/mysql-test/lsan.supp b/mysql-test/lsan.supp
index b3dcaa36671..916a057d8d8 100644
--- a/mysql-test/lsan.supp
+++ b/mysql-test/lsan.supp
@@ -28,3 +28,7 @@ leak:Perl_savesharedpv
 leak:Perl_Slab_Alloc
 leak:Perl_newUNOP_AUX
 leak:/usr/bin/perl
+leak:/bin/bash
+leak:/usr/bin/sed
+leak:/usr/bin/python2.7
+leak:/usr/bin/python3.8
```
as in Percona Server 8.0

Reply:
Comment in the "Conflicts resolved" commit is misleading
Should be
```
***
VERSION raised to "5.7.31-34".
PERCONA_INNODB_VERSION in univ.i raised to "34".
```
instead of 
```
***
VERSION raised to "5.7.31-34.0".
PERCONA_INNODB_VERSION in univ.i raised to "34.0".
```